core/utils/time_test.cc

```diff
@@ -8,6 +8,12 @@ TEST(RdtscTest, NonDecreasing) {
   ASSERT_LE(a, b) << "Time stamp counter should not decrease.";
 }
 
+TEST(TscToUs, Frequency) {
+  ASSERT_NE(0, tsc_hz) << "tsc_hz has not been initialized";
```

Comment:
Will tsc_hz be 0 if TscHzSetter() hasn't been called?  Or will it be some undefined value?